src/ccl_power.c

```diff
@@ -475,12 +472,18 @@ void ccl_cosmology_compute_linear_power(ccl_cosmology* cosmo, ccl_f2d_t *psp, in
     cosmo->computed_linear_power = true;
 }
 
+void ccl_cosmology_compute_nonlin_power_from_f2d(ccl_cosmology *cosmo,
```

Comment:
Do we actually need this function, or can we do the copy right in the function below?

Reply:
this is what I'd want to expose in python

Reply:
No, good point. Let me do that quickly.

Reply:
I'd rather not do this here, though. We'll be forced to do it when we try to get things directly from CAMB, and that seems like an isolated issue to me.